client/components/forms/form-phone-media-input/index.jsx

```diff
@@ -17,6 +17,15 @@ export default class extends React.Component {
 		isError: false,
 	};
 
+	constructor( props ) {
+		super( props );
+		this.focus = this.focus.bind( this );
```

Comment:
Why do it this way? Isn't that just an alternative syntax for: 
```
focus = () => {...
```

Reply:
Yeah, yeah. Ok. :-D